packages/insured-bridge-relayer/test/Relayer.ts

```diff
@@ -783,6 +836,68 @@ describe("Relayer.ts", function () {
       await relayer.checkForPendingRelaysAndDispute();
       assert.isTrue(lastSpyLogIncludes(spy, "Disputed pending relay"));
     });
+    it("Two pending relays with invalid relay data, one dispute succeeds, one fails", async function () {
```

Comment:
these tests are great. as everything else passes without need to be changed it implies that we have not broken anything else.